common/starbase_buildings/hfe_starbase_building.txt

```diff
@@ -70,9 +70,16 @@ hfe_starbase_building_drone_logistics = {
 		country_resource_max_energy_add = 1000
 		country_resource_max_minerals_add = 1000
 		country_resource_max_alloys_add = 200
-		country_naval_cap_add = 10
-		country_starbase_capacity_add = 0.5
+		country_naval_cap_add = 20
+		country_starbase_capacity_add = 1
```

Comment:
Too much

Reply:
Not true. The HFE types are ALWAYS over their Fleet Limit.